Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Relay/Controller.mo

```diff
@@ -164,38 +155,42 @@ equation
           fillPattern=FillPattern.Solid,
           fillColor={175,175,175},
           textString="Relay"),
-        Line(points={{-70,24},{-34,24},{-34,58},{38,58},{38,24},{66,24}}, color
-            ={28,108,200})}), Diagram(
+        Line(points={{-70,24},{-34,24},{-34,58},{38,58},{38,24},{66,24}}, color=
+             {28,108,200})}), Diagram(
         coordinateSystem(preserveAspectRatio=false)),
 Documentation(info="<html>
 <p>
 This block generates a real control output <code>y</code>, a
-boolean relay switch output <code>yOn</code>, and the control error
+boolean relay switch output <code>yOn</code>, an input difference 
+<code>yDif</code>, and a control error
 <code>yErr</code>. They are calculated as below:
 </p>
 <ul>
 <li>
-if the parameter <code>reverseActing = false</code>
+<code>yErr = u_m - u_s</code>,
+</li>
+<li>
+if the parameter <code>reverseActing = true</code>
 <ul>
 <li>
-<code>yErr = u_m - u_s</code>,
+<code>yDif = - yErr</code>,
 </li>
 </ul>
 </li>
 <li>
 else
 <ul>
 <li>
-<code>yErr = u_s - u_m </code>,
+<code>yDif = yErr</code>,
 </li>
 </ul>
 </li>
 <li>
-if <code>yErr &lt; -deaBan</code> and <code>trigger</code> is <code>true</code>,
+if <code>yDif &lt; -deaBan</code> and <code>trigger</code> is <code>true</code>,
 then <code>y = yHig</code>, <code>yOn = true</code>,
 </li>
 <li>
-else if <code>yErr &gt; deaBan</code> and <code>trigger</code> is <code>true</code>,
+else if <code>yDif &gt; deaBan</code> and <code>trigger</code> is <code>true</code>,
```

Comment:
Note that in current implementation, if `yDiff` (the input to instance `hys`) > `deaBan`, then, `hys.y = true`, `yOn=true` and `y=yHig`. This is opposite to the documentation here.
Which one is right?

Reply:
Sorry for confusion and actually I confused myself with the signs. I have updated the model doc.